pkg/ee/cluster-backup/user-cluster/velero-controller/resources/cluster_backup.go

```diff
@@ -51,6 +51,9 @@ const (
 
 	version       = "v1.14.0"
 	pluginVersion = "v1.10.0"
+
+	backupOrigin = "kkp-backend"
```

Comment:
Is there also a KKP frontend? Normally the term "backend" isn't really used in the KKP world.

Reply:
Then I'd still call it "controller" or "kkp-controllers". I don't know why I have such a strong dislike for the word "backend" here, maybe because there could be more than one "backend" 😅